counterpartylib/lib/micropayments/control.py

```diff
@@ -0,0 +1,729 @@
+# coding: utf-8
+# Copyright (c) 2016 Fabian Barkhau <fabian.barkhau@gmail.com>
+# License: MIT (see LICENSE file)
+
+
+import copy
+import pycoin
+from . import validate
+from . import exceptions
+from micropayment_core import util
```

Comment:
you might also want to import this file as `mpc_util` instead of `util` as there is another `util.py` and whenever I see `util.` I am automatically thinking it's something in `util.py`....